github/src/webhook.rs

```diff
@@ -35,7 +35,7 @@ impl Webhook {
                 "[webhook {}] No secret specified; signature ignored",
                 self.delivery_id
             );
-            return true;
+            return false;
```

Comment:
I don't know how to feel about this since its completely ok to not configure a webhook secret

Reply:
I think not using the web book secret is something to not support, otherwise it’s easy to not know it’s configured incorrectly